doc/release-notes/820-non-ascii-chars-in-search.md

```diff
@@ -0,0 +1,8 @@
+### Search with non-ascii characters
+
+Many languages include characters that have close analogs in ascii, e.g. (á, à, â, ç, é, è, ê, ë, í, ó, ö, ú, ù, û, ü…). This release changes the default Solr configuration to allow search to match words based on these associations, e.g. a search for Mercè would match the word Merce in a Dataset, and vice versa. This should generally be helpful, but can result in false positives.,e.g. "canon" will be found searching for "cañon". 
+
+## Upgrade Instructions
+
+1. You will need to replace or modify your `schema.xml` and restart solr. Re-indexing is required to get full-functionality from this change - the standard instructions for an incremental reindex could be added here.
```

Comment:
When PR #7645 gets merged (in QA), this PR will need to be updated, as these release notes should be merged with the release notes included in that Solr update PR.

Reply:
Here are the release notes from the Solr upgrade PR.

> ### Solr Update
> 
> With this release we upgrade to the latest available stable release in the Solr 8.x branch. We recommend a fresh installation of Solr 8.8.1 (the index will be empty)
> followed by an "index all".
> 
> Before you start the "index all", Dataverse will appear to be empty because
> the search results come from Solr. As indexing progresses, partial results will 
> appear until indexing is complete.
> 
> 
> See http://guides.dataverse.org/installation/prerequisites.html#installing-solr
> 
> [for the additional upgrade steps section]
> 
> Run the script updateSchemaMDB.sh to generate updated solr schema files and preserve any other custom fields in your Solr configuration.
> For example: (modify the path names as needed)
> cd /usr/local/solr-8.8.1/server/solr/collection1/conf
> wget https://github.com/IQSS/dataverse/releases/download/v5.4/updateSchemaMDB.sh
> chmod +x updateSchemaMDB.sh
> ./updateSchemaMDB.sh -t .
> 
> See http://guides.dataverse.org/en/5.4/admin/metadatacustomization.html?highlight=updateschemamdb for more information.

Reply:
@djbrooke in tech hours, @scolapasta suggested moving this PR forward with the note to REMOVE this "Upload Instructions" section, as it a duplicate of the steps already included in the Solr upgrade PR #7645.

Reply:
@mheppler great, thanks, I'll leave a note in this PR at the top of the notes so I don't forget, in case something slips and we don't get this into the release we expect to.